pkg/controller/tainteviction/taint_eviction.go

```diff
@@ -105,7 +116,7 @@ func deletePodHandler(c clientset.Interface, emitEventFunc func(types.Namespaced
 	return func(ctx context.Context, args *WorkArgs) error {
 		ns := args.NamespacedName.Namespace
 		name := args.NamespacedName.Name
-		klog.FromContext(ctx).Info("NoExecuteTaintManager is deleting pod", "pod", args.NamespacedName.String())
+		klog.FromContext(ctx).Info(fmt.Sprintf("%s is deleting pod", TaintEvictionController), "pod", args.NamespacedName.String())
```

Comment:
ditto (structured-logging convention)

nit: `args.NamespacedName` is sufficient